integration-cli/requirements.go

```diff
@@ -149,9 +149,20 @@ var (
 				 */
 				return false
 			}
+
+			// We need extra check on redhat based distributions
+			if f, err := os.Open("/sys/module/user_namespace/parameters/enable"); err == nil {
+				b := make([]byte, 1)
+				_, _ = f.Read(b)
+				if string(b) == "N" {
```

Comment:
It's just a test so we don't really care, but `return string(b) != "N"` should work ;-)